### DISCUSSION TITLE
There is no warning message before running the project even though the project contains error.

### DISCUSSION THREAD

**Discussion by linette-zyy** [Microsoft/PTVS#7336] [Enchancements]
**Environment**
![image](https://user-images.githubusercontent.com/59192491/119951195-4400bf80-bfce-11eb-97d5-337345270688.png)

**Steps to Reproduce**
1. Checked the "Prompt before running when errors are present" in tools/options/python/debugging
2. Create a python application project
3. Type print("hello python" in default opened file PythonApplication1.py and save it
4. Start debugging F5 for CTRL + F5
5. Observe

**Expected behavior**
Expect a warning message box like
![image](https://user-images.githubusercontent.com/59192491/119951427-85916a80-bfce-11eb-89c3-187bec57ade3.png)

**Actual result**
There is no warning message.
![Without Prompt](https://user-images.githubusercontent.com/59192491/119952537-b32ae380-bfcf-11eb-9da3-e7a99522a133.gif)

**Comment by greazer** [comment]
This doesn't meet the bar for fixing right now. The user can see whether there are errors before they press F5.

**Comment by ijoosong** [comment]
@judej This could be a potential intern project?

**Comment by StellaHuang95** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
